### DISCUSSION TITLE
Trilium-Py script to publish file to sync server

### DISCUSSION THREAD

**Discussion by jwhonce** [TriliumNext/Trilium#4086] [Show and tell]
I find writing JavaScript in `code` easy / faster than in a trilium note. AutoComplete is my friend. I also wanted to take [trilium-py](https://github.com/Nriver/trilium-py) for a spin.  The script matches filename to note title. I wrote this as a CLI for ease in integrating into IDE's.

Python code can be found at: https://github.com/jwhonce/trilium-addons/blob/main/pushToNote/pushToNote.py

Have fun! As always this is a WIP, there are TODO's and potential improvements to be made.

@zadam Thanks for a great tool
@Nriver Thanks for the Python bindings

**Comment by Nriver** [comment]
Looks great!
